### PR TITLE
Update base-no.yaml

### PR DESCRIPTION
**Pull request by NiceManiac**
Translated the Steam store page completely to Norwegian

### CODE REVIEW

File: translations/base-no.yaml [hunk 1]
```diff
@@ -30,59 +30,90 @@ steamPage:
     longText: >-
         [img]{STEAM_APP_IMAGE}/extras/store_page_gif.gif[/img]
 
-        shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
-        Upon delivering the requested shapes you will progress within the game and unlock upgrades to speed up your factory.
+        #shapez.io is a game about building factories to automate the creation and processing of increasingly complex shapes across an infinitely expanding map.
```
Comment (MizardX) [inline translations/base-no.yaml:33]: Comments inside multi-line strings are not allowed in YAML. They are interpreted as part of the string.
Please either remove them, or move them outside the string.
```
#longText: >-
#  original english
longText: >-
  translated text
```

**Comment by NiceManiac** [comment]
Updated to be the right formatting for YAML, forgot about mulitiline commenting

**Comment by tobspr** [comment]
Could you remove the comment at the top though? :)

**Comment by NiceManiac** [comment]
removed comments of orignal text

**Comment by tobspr** [comment]
ty!

**Pull request merged**